ndk-make.sh

```diff
@@ -1,60 +1,50 @@
 #!/bin/sh
-set -e
-echo "starting time: `date`"
-
-cd jni/deltachat-core-rust
 
-# to setup the toolchains (from https://medium.com/visly/rust-on-android-19f34a2fb43 )
-# run `scripts/install-toolchains.sh`.
+# If you want to speed up compilation, you can run this script with your
+# architecture as an argument:
+#
+#     ./ndk-make.sh arm64-v8a
+#
+# Possible values are armeabi-v7a, arm64-v8a, x86 and x86_64.
+# You should be able to find out your architecture by running:
+#
+#     adb shell uname -m
+#
+# or:
+#
+#     adb shell cat /proc/cpuinfo
+#
+# The values in the following lines mean the same:
 #
-# Currently ndk20b is minimum required version
-# (newer versions will likely work, however, are not tested and not used in offial releases,
-# in general, changes on the ndk-version should be done with care)
+# armeabi-v7a, armv7 and arm
+# arm64-v8a, aarch64 and arm64
+# x86 and i686
+# (there are no synonyms for x86_64)
 #
-# after that, add PATH_TO_NDK/toolchains/llvm/prebuilt/HOST/bin to your $PATH
-# and add the correct clang-linkers to `~/.cargo/config`:
-# ```
-# [target.armv7-linux-androideabi]
-# linker = "PATH_TO_NDK/toolchains/llvm/prebuilt/HOST/bin/armv7a-linux-androideabi16-clang"
-# [target.aarch64-linux-android]
-# linker = "PATH_TO_NDK/toolchains/llvm/prebuilt/HOST/bin/aarch64-linux-android21-clang"
-# [target.i686-linux-android]
-# linker = "PATH_TO_NDK/toolchains/llvm/prebuilt/HOST/bin/i686-linux-android16-clang"
-# [target.x86_64-linux-android]
-# linker = "PATH_TO_NDK/toolchains/llvm/prebuilt/HOST/bin/x86_64-linux-android21-clang"
-# ```
-# then, the following should work:
-# (If you want to speed up compilation, have look at the comment in ndk-make-fast.sh.)
+#
+# If you put this in your .bashrc, then you can directly build and deploy DeltaChat from the jni/deltachat-core-rust directory by typing nmake():
+# nmake() {sh -c 'cd ../..; ./ndk-make.sh arm64-v8a && ./gradlew installFatDebug; notify-send "install finished" }
```

Comment:
```suggestion
# If you put this in your .bashrc, then you can directly build and deploy DeltaChat from the jni/deltachat-core-rust directory by typing nmake():
# (you have to replace arm64-v8a by your architecture)
# nmake() {sh -c 'cd ../..; ./ndk-make.sh arm64-v8a && ./gradlew installFatDebug; notify-send "install finished"' }
```
esp. missing `'` (was probably me who forgot this when writing this text)

Reply:
Pushed a change that starts a subshell with `(...)`